docs/interface.rst

```diff
@@ -262,6 +262,22 @@ Each item in the ``dict`` maps a relpath to a list of author-names.
 The author-names correspond to the device that conflicts with this file.
 There will also be a file named like ``<relpath>.conflict-<author-name>`` in the magic-folder whose contents match those of the conflicting remote file.
 
+.. _api_resolve_conflict:
+
+POST ``/v1/magic-folder/<folder-name>/resolve-conflict``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Ask for a particular resolution to a conflicted file.
+The body is a JSON object containing the keys:
+
+* ``relpath``: the file name inside the folder
+* ``take``: ``"mine"`` or ``"theirs"`` indicating whose version to take
+* ``use``:  a Participant name indicating whose version to take
+
+Note that you can use only one of ``take`` or ``use``, and that ``take="theirs"`` only works when there is exactly one other conflict.
```

Comment:
What actually happens (or is expected to happen) if both are specified? Will users of the API get a 500 error? A response containing an error message in the body? Something else?

Reply:
You get an `_InputError` which is rendered as a BAD_REQUEST like all other input validation errors in the API.